migrations/2020-02-25-215149_create_matches/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE matches (
```

Comment:
I think it should be idempotent, so you need to check if the table already exists

Reply:
`CREATE TABLE  IF NOT EXISTS ...`

Reply:
Ahhh I see. Thanks. 👍